mmv1/templates/terraform/examples/target_grpc_proxy_basic.tf.erb

```diff
@@ -1,3 +1,4 @@
+[START cloudloadbalancing_target_grpc_proxy_basic]
```

Comment:
```suggestion
# [START cloudloadbalancing_target_grpc_proxy_basic]
```

All Start/End tags need to be commented out, otherwise Terraform thinks they are actual HCL. Without commenting, these example blocks are failing during tests

Reply:
Thank you. Updated the tags now!